app/src/layouts/appLayout/projectSidebar/projectSidebar.jsx

```diff
@@ -164,39 +125,60 @@ export class AppSidebar extends Component {
       }),
     );
 
-    return topItems;
-  };
+    const topSidebarItemsMap = topItems.map(
```

Comment:
Why it is considered as `Map`? It is an array with objects.
Let's name it just as `topSidebarItems`.

Reply:
This was necessary before refactoring to avoid eslint no-shadow issue. 
Renamed.